extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -19,6 +19,7 @@
       - [Authorised Response](#authorised-response)
       - [Action Response](#action-response)
     - [Klarna payment](#klarna-payment)
+    - [Affirm payment](#affirm-payment)
```

Comment:
Pls update the content section too.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/785/commits/4a87363942f5e05bdd76fe8bec041ea4b6fb6a22

Reply:
Step 7 is still not updated. Do you know you can run `npm run doctoc` and it will update table of content (toc) for you?

Reply:
Thanks for telling
https://github.com/commercetools/commercetools-adyen-integration/pull/785/commits/f9cde4b4534ef910abcd5cacc737a6dbf09ba6d4